incl/minishell.h

```diff
@@ -50,6 +52,7 @@ typedef struct s_data
 	char	**input_split;
 	int		orig_fdin;
 	int		orig_fdout;
+	int		hd_fd;
```

Comment:
Added heredoc file descriptor here, so I can close it in exit_minishell  even if the heredoc process is interrupted with SIGINT (Ctrl + C).